runtime/compiler/control/rossa.cpp

```diff
@@ -1923,6 +1923,14 @@ onLoadInternal(
 
    jitConfig->jitAddNewLowToHighRSSRegion = jitAddNewLowToHighRSSRegion;
 
+#ifndef J9VM_OPT_JITSERVER
```

Comment:
This is not entirely correct. We want to use this facility at the client too. `#ifndef J9VM_OPT_JITSERVER` means that JITServer support is completely eliminated from the code. On Linus, `J9VM_OPT_JITSERVER` is defined by default.